lib/ingestors/ingestor_event_rest.rb

```diff
@@ -401,6 +417,226 @@ def process_elixir(url)
       end
     end
 
+    def process_vu(url)
+      # execute REST request
+      results = get_JSON_response url
+      data = results.to_h['results']
+
+      # extract materials from results
+      unless data.nil? or data.size < 1
+        data.each do |item|
+          begin
+            # create new event
+            event = Event.new
+
+            # extract event details from
+            attr = item
+            event.title = attr['title']
+            event.url = attr['url'].strip unless attr['url'].nil?
+            event.organizer = attr['org']
+            event.description = convert_description attr['description']
+            event.start = attr['startdt']
+            event.end = attr['enddt']
+            event.venue = attr['location']
+            event.city = 'Amsterdam'
+            event.country = 'The Netherlands'
+            event.source = 'VU'
+            event.online = attr['online_event']
+            event.contact = attr['orgurl']
+            event.timezone = 'Amsterdam'
+
+            # array fields
+            event.keywords = []
+            attr['categories_arr'].each { |category| event.keywords << category['name'] } unless attr['categories'].nil?
```

Comment:
one is categories_arr, the other is categories. Is that intentional? Perhaps a comment would be nice, or if they should be the same you can use safe navigation